docs/pip.md

```diff
@@ -270,7 +270,7 @@ Cachi2 looks for the name and version of your project in the following project f
 * [setup.cfg](#setupcfg)
 * [setup.py](#setuppy)
 
-If Cachi2 fails to resolve the necessary metadata, it will refuse to process the project. This may change in the future.
+If Cachi2 fails to resolve the necessary metadata from project files, it will try to resolve the name from git repository origin url and package subpath (version is optional). If git repository does not exists, then it will refuse to process the project.
```

Comment:
I think describing what happens for name and version separately would be clearer

> If Cachi2 fails to resolve the project name, it will generate a name based on the git repository origin url (and package subpath, if any). If Cachi2 fails to resolve the version, it will omit the version.

IMO we don't need to mention what happens if the directory is not a git repo.

And please keep the line length roughly aligned with the rest of the doc :pray: